src/fe/inf_fe_static.C

```diff
@@ -178,6 +178,8 @@ Real InfFE<Dim,T_radial,T_map>::shape(const FEType & fet,
   compute_shape_indices(fet, inf_elem_type, i, i_base, i_radial);
 
   //TODO:[SP/DD]  exp(ikr) is still missing here!
+  // but is it intended?  It would be probably somehow nice, but than it would be Number, not Real !
+  // --> thus it would destroy the interface...
```

Comment:
This I couldn't tell you about... but I was under the strong impression that infinite elements were supposed to work even without --enable-complex!

Reply:
... I will check that. 
But if we use complex numbers, the 'full' shape of infinite elements include a complex oscillation, so it becomes complex.